server/utils/journeyTypeFromArtifact.ts

```diff
@@ -0,0 +1,9 @@
+import { FormArtifact, JourneyType } from '../@types/ui'
+import isAssessment from './assessments/isAssessment'
+import { isPlacementApplication } from './placementRequests/isPlacementApplication'
+
+export const journeyTypeFromArtifact = (artifact: FormArtifact): JourneyType => {
+  if (isAssessment(artifact)) return 'assessments'
+  if (isPlacementApplication(artifact)) return 'placement-applications'
```

Comment:
Wonder if it might be worth documenting how this works, it could be a bit confusing as Assessments and PlacementApplications both has applicationIDs

Reply:
I believe that assessments have the actual application attached as opposed to the ID so the order doesn't matter but that could change at some point and the order would matter